packages/flutter_tools/test/commands.shard/permeable/upgrade_test.dart

```diff
@@ -315,7 +334,7 @@ class FakeUpgradeCommandRunner extends UpgradeCommandRunner {
   Future<void> upgradeChannel(FlutterVersion flutterVersion) async {}
 
   @override
-  Future<bool> attemptReset(FlutterVersion flutterVersion, String newRevision) async => alreadyUpToDate;
+  Future<void> attemptReset(String newRevision) async {}
```

Comment:
the return value was a mistake that should have been cleaned up in my previous PR.